lib/validator.js

```diff
@@ -330,6 +330,13 @@ validator.validatePutRequest = function(req, schema) {
         throw new Error('Invalid query. No schema for ' + req.table);
     }
 
+    var missingColumn = Object.keys(req.attributes).find(function(attrName) {
```

Comment:
Could we separate this into its own method (and then call it from this one), so that we can reuse it in the Cassandra module and reduce code duplication?